### DISCUSSION TITLE
Codespace price github team

### DISCUSSION THREAD

**Discussion by Shizanit** [community/community#137737] [Codespaces] [answered]
### Select Topic Area

General

### Body

In github team plan each account gets free 180 hours of codespace?

**Comment by LiteBrite82** [comment, marked as answer]
Hi @Kitanit 
The answer is  kind of complicated. 
TL;DR: 
- The Free, Team, and Enterprise plans for organization and enterprise accounts do not include any free use of GitHub Codespaces.
- The Free and Pro plans for personal accounts include free use of GitHub Codespaces up to a fixed amount of usage every month.

So, members of the team can use the hours associated with their personal account. But Codespaces owned and run by the team account are all paid.  It all depends on the ownership settings for the team. 

[About billing for GitHub Codespaces](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#codespaces-pricing)

**Comment by riowaller** [comment]
Thanks for asking this question.  I just started using codespaces in github organization that I use to run my classes for coding at the community college system in California.   After researching it this summer I thought the users (students) would be under the time limits for the organization, but today I realized that was set to user, so it is using their account codespace limits.  This came up because I had one student report they were unable to access their codespace because they had used the 120 limit.  

So I'm not sure if i should change it to be owned by the organization, but I don't think I actually can, based on what I saw in the billing section and @LiteBrite82 comment above.

Can student grade their github account to education and does it give me more hours? Like 180, I looked on the github education site it listed 60 hours...so I'm like...that is less than they have now?

Thanks for your input and/or advice.  

Rio Waller 
Fresno City College
